src/nvim/ex_docmd.c

```diff
@@ -159,18 +159,18 @@ static char dollar_command[2] = { '$', 0 };
 
 static void save_dbg_stuff(struct dbg_stuff *dsp)
 {
-  dsp->trylevel       = trylevel;             trylevel = 0;
-  dsp->force_abort    = force_abort;          force_abort = false;
-  dsp->caught_stack   = caught_stack;         caught_stack = NULL;
-  dsp->vv_exception   = v_exception(NULL);
-  dsp->vv_throwpoint  = v_throwpoint(NULL);
+  dsp->trylevel = trylevel;             trylevel = 0;
```

Comment:
this breaks the "tabular" format of the second statement. Doesn't seem to be many cases of this, so could just correct it manually (or just move them to new lines, either is fine).